client/src/components/QA/QaQuestion.jsx

```diff
@@ -50,7 +50,7 @@ const QaQuestion = ({question}) => {
         <div>
           <h3><strong>Q: {question.question_body}</strong></h3>
           <div>
-            <h4>{question.asker_name}, {moment(question.question_date).format('MMMM Do YYYY')} | Helpful? Yes({question.question_helpfulness}) | <button onClick={() => {setAnswerModal(true)}}>Add Answer</button> | Report</h4>
+            <h4>{question.asker_name}, {moment(question.question_date).format('MMMM Do YYYY')} | Helpful? Yes({question.question_helpfulness}) | <span className="addAnswerBtn" onClick={() => {setAnswerModal(true)}}>Add Answer</span> | Report</h4>
```

Comment:
same thing here (camel case), its fine if you're staying consistent with how the db names its values.